src/index.html

```diff
@@ -1,13 +1,50 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
+    <div class="card" data-qa="card">
+      <div class="card__image-block">
+        <img
+         class="card-image"
+         src="images/imac.jpeg"
+         alt="iMac"
+        >
+      </div>
+      <div class="card__text-name">
+        Моноблок APPLE A1419 iMac <br>27" Retina 5K (MNED2UA/A)
+      </div>
+      <div class="card__id">Код товара: 195434</div>
+      <div class="card__rate-block">
+        <div class="stars card__stars--active--4">
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+        </div>
+        <span>Отзывов: 5</span>
+      </div>
+      <div class="card__price">
+        <span class="price-text">Цена:</span>
+        <span class="price-summ">69 999 грн</span>
+      </div>
```

Comment:
```suggestion
      <div class="card__price">
        <span class="card__price-text">Цена:</span>
        <span class="card__price-summ">69 999 грн</span>
      </div>
```
don't make 1 line of text a separate BEM-block